src/index.html

```diff
@@ -13,6 +13,144 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section>
+      <div class="stars stars--0">
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
```

Comment:
The 'div' tag is being used here. It would be better to use a more semantic tag, like 'ul' for the list of stars, with each star being an 'li' element. This would improve the semantics of your HTML and help with accessibility and SEO.